_targets/build.project.armv7a7-imx6ull

```diff
@@ -19,15 +19,15 @@ export PSH_DEFUSRPWDHASH="0B1ANiYi45IhxkfmUW155/GBd4IRE="
 # Ports configuration
 #
 export PORTS_BUSYBOX=y
-export PORTS_PCRE=n
-export PORTS_OPENSSL=n
-export PORTS_LIGHTTPD=n
+export PORTS_PCRE=y
```

Comment:
IMHO these should probably be changed in `imx6ull-evk` project (the _targets/* files are shared between projects so changing default values might have unexpected consequences in other projects (like DTR))